### PR TITLE
Wrote the function to calculate L from n_dens and n_particles.

### PR DESCRIPTION
**Pull request by madilynpaul**

### CODE REVIEW

**Review by chrisjonesBSU — APPROVED** [review]

**Comment by erjank** [comment]
Something complicating this PR is the r_factor variable- what is it for? And separately, we don't need to allow for disks of different size for now - let's assume the radius for all disks is 0.5

**Comment by chrisjonesBSU** [comment]
Yeah, I don't think the `r_factor` variable is needed, we have a different parameter to set radius. It looks like this PR removes the `r_factor` variable completely.  I guess we should remove it from the doc strings as well.

File: MCMC/simulation.py [hunk 1]
```diff
@@ -11,10 +11,9 @@
...
 
 class Simulation:
-    def __init__(self, n_density=0.5, r=0.1, r_factor=5, kT=1.0, r_cut=1, max_trans=0.2, write_freq=5,
+  def __init__(self, n_density=0.5, n_particles=5, r=0.1, kT=1.0, r_cut=1, max_trans=0.2, write_freq=5,
                  energy_func=inverse_distance_energy):
         """
-
         :param n_density: Number density.
         :param r: Disk radius.
         :param r_factor: Box length is r * r_factor.
```
Comment (chrisjonesBSU) [inline MCMC/simulation.py:19]: We can delete this line since this PR removes the r_factor parameter.

Comment (chrisjonesBSU) [inline MCMC/simulation.py:19]: We can delete this line since we are removing the `r_factor` parameter.

**Pull request merged**